content/user/ui/settings/Groups.md

```diff
@@ -1,74 +1,65 @@
+
+### Creating an Organization Group from Organization Settings
+
+Groups created from the Organization Settings interface only impact the roles and permissions of the organization in which they are created. 
+
+> For example, let’s say that you have two organizations in Contrast: OrgA and OrgB. When you log into OrgB and create a group in OrgB, the permissions and roles that you define affect only OrgB users and OrgB applications.
+
+To create a group, you must have an Admin role in the organization in which you want to create the group. Once you log in as an Admin, navigate to the Organization Settings interface and then to the Groups interface.
+
+Begin creating your group by defining the level of access that you want the group to provide. Groups define access as a pairing of application(s) and role. In the applications field, select a single application, multiple applications or all applications. Then select the role you want to provide to those applications. The available roles are Admin, Rules Admin, Edit, View and No Access. 
+
+For more information, read the article on [Roles](https://docs.contrastsecurity.com/user_tsguideset.html#roles).
+
+>Example: You can select App1 and App2 and the role Admin in a single line. This means the group provides users the Admin role for App1 and App2.
+
+A single group can also provide more than one role assignment. Select **Add Access** to add another line in the Group definition, and define a separate application and role assignment. 
+
+> From the previous example: Your group provides Admin access to App1 and App2. You can add a second access line that provides Edit access to App3 and App4.
+
+Once you define the access that you want the group to provide, you can assign the users that you want to receive this access. You may select any user that is in your organization, other than yourself or any user that is considered a Guest user. When you’re finished, just hit save to create your access group.
+
+To read more about Guest users, refer to the section under [About Access Control](https://docs.contrastsecurity.com/user_tsguideset.html#access).
+
+### Editing an Organization Group from Organization Settings
+
+To edit an existing organization, go to the Edit interface by selecting the organization’s name.  Within the interface, you can edit any of the fields that are available when creating a group. This includes the name of the group, the users assigned to the group and any access provided by the group. 
+
+Default groups are the single exception here. You can add or remove users from a Default group, but you can’t edit any of the applications or the roles to which the group provides access.
+
+> Example: If your access group has three users and provides Admin access for App1 and App2, as well as Edit access for App3 and App4, I can edit the group to:
+* remove User3 and add User4 
```

Comment:
In all of the earlier bullet point sections in this article, we are capitalizing the first word.